frontend/src/app/model/table-data-daily.ts

```diff
@@ -0,0 +1,13 @@
+import {CovidCasesDaily} from './covid-cases-daily';
+import {CovidDeathsDaily} from './covid-deaths-daily';
+import {HospitalBedsDaily} from './hospital-beds-daily';
+
+export class TableDataDaily {
```

Comment:
Should be renamed. Do **not** give a class a name which stands for what it is used in a particular case. Give it a name that tells what it is.